semver.md

```diff
@@ -91,7 +91,7 @@ incompatible changes are introduced to the public API. It MAY also include minor
 and patch level changes. Patch and minor version MUST be reset to 0 when major
 version is incremented.
 
-1. A pre-release version MAY be denoted by appending a hyphen and a
```

Comment:
Would it be clearer to simply change this to:

> A pre-release version is denoted by...

Reply:
@Haacked, _Is_ is not defined in [RFC 2119](http://tools.ietf.org/html/rfc2119). Standards should use standardized semantics to define themselves, yes?

Reply:
I've been away from this for quite a bit, but trying to re-enter this discussion in a meaningful way. 

Outside the world of this spec there is a de facto convention some pre-release packages use: a major version of "0"!    I believe this should be discouraged, rather than prohibited, as there's no way to change the past.   That's why I suggested rule #9 be updated to the "SHOULD" in the first place.   

I hope this addresses @jwdonahue's question.